docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,293 @@
+
+   - For a DM document, create a comment on the RFC confirming the changes and link to the ``https://<handle>.lsst.io/v/RFC-<n>`` landing page for the release branch.
+     You don't need to create intermediate DocuShare versions for the DM CCB.
+
+   Repeat this step for each round of CCB feedback.
+
+5. When the CCB approves document you create a release:
+
+   1. Commit to the head of the release branch with the following changes:
+
+      - Update `document's change record <https://lsst-texmf.lsst.io/lsstdoc.html#document-preamble>`_.
+        The Project librarian or DM release manager, through the CCB, determines the document's semantic version.
+      - Remove the ``draft`` option from the document class.
+      - Set the ``\date`` command using a YYYY-MM-DD format.
+
+   2. :ref:`Create a new DocuShare upload <ccd-docushare-upload>`.
+
+   3. Once the Project librarian or DM documentalist has uploaded the document and made it the new preferred version, create a :ref:`semantic version tag <ccd-semantic-tag>` at the same commit as the DocuShare tag:
+
+      .. code-block:: bash
+
+         git tag -a v<major>.<minor>
+         git push --tags
+
+      In your command, replace ``<major>.<minor>`` with the semantic version.
+
+      Format the Git tag message as:
+
+      .. code-block:: text
+
+         v<major.minor>
+
+         https://docushare.lsst.org/docushare/dsweb/Get/Version-<...>
+
+      The URL should point to the DocuShare version (same as the DocuShare tag).
+
+   4. Backport the amendment commits made on the release branch back to the ``master`` branch:
+
+      1. Create a user branch from the ``master`` branch:
+
+         .. code-block:: bash
+
+            git checkout master
+            git checkout -b u/<username>/v<major>.<minor>-backport
+
+      2. Cherry-pick commits from the release branch onto the new backport branch.
+         For example:
+
+         .. code-block:: bash
+
+            git cherry-pick <commit-sha>
+         
+         Also ensure that the document's change record is updated, but the ``draft`` option must remain in the document class command.
+
+      3. Push the backport branch to GitHub for continuous integration validation, rebase, and merge to master.
+         For example:
+
+         .. code-block:: bash
+
+            git checkout master
+            git pull
+            git checkout u/<username>/v<major>.<minor>-backport
+            git rebase -i master
+            git push -u  # --force
+            git checkout master
+            git merge --no-ff u/<username>/v<major>.<minor>-backport
+            git push
+
+.. _ccd-hotfix:
+
+Hotfixing a released document
+=============================
+
+The procedure above (:ref:`ccd-release`) describes how to make a new version of a document from the ``master`` branch.
+Sometimes it is necessary to hotfix a released document to fix a typo or make a similar minor change.
```

Comment:
> ...whilst other edits of this document are ongoing. ?

I think that if you are making a change to a document that has no other changes being made to it you can use the normal process.